code/modules/mob/living/carbon/human/species_types/zombies.dm

```diff
@@ -17,6 +17,7 @@
 		"appendix" = /datum/internal_organ/appendix,
 		"eyes" = /datum/internal_organ/eyes
 	)
+	death_message = "seizes up and falls limp..."
 	///Sounds made randomly by the zombie
```

Comment:
isnt this on the parent type too

Reply:
> isnt this on the parent type too

kind of its  "seizes up and falls limp, their eyes dead and lifeless..." but they were already dead and lifeless, its more thematic